### ISSUE TITLE
Words beginning with a Turkish character

### ISSUE THREAD

**Issue by Emperyal** [zioth/dokuwiki-autolink4#9]
I released that there's a parsing issue with Turkish characters like ı, İ, Ş, ş at the beginning of the word or word group. For instance, when I add words like Iğdır, İsrail, Şekerpare the plugin doesn't link them. How can I fix this? Any solution would be appreciated.

**Comment by Emperyal** [comment]
Okay, I just noticed that this problem only occurs when I used regex in the words starting with a Turkish character to match ones that has suffixes. The plugin works well in words that do not begin with a Turkish character. When it comes to words starting with a Turkish character the plugin only links the exact string and doesn't support regular expressions unlike other words. So this makes word matching unflexible for words beginning with a Turkish character.
